### ISSUE TITLE
Make "Edit" from detail page spawn an edit-only composer view

### ISSUE THREAD

**Issue by jonpmarino** [MapStory/mapstory-meta#1390]
When a user clicks "edit" from the layer detail page on a geogig-enabled layer, it should launch an edit-mode on that layer. Edit mode is just the "edit layer" functions from #1382 without the rest of composer enabled (chapters, pins, boxes, legend, etc). Essentially, it enables a user to add/delete/modify features and see the edit history. Once the user is done editing, they should be able to "go back to layer detail" or "launch a new mapstory". Launching a new mapstory results in composer with that layer added as layer 1 in chapter 1 (just like clicking 'compose a maptory' from layer detail page)

**Comment by davisc** [comment]
@jonpmarino moving this down the line as we need to complete composer before we try to implement the exclusive edit interface.

**Comment by Coop56** [comment]
This is done but has two errors left on it.  @JWileczek and I worked through a bunch of errors to get this working, but there's a few remaining that you may need to look at @davisc.

**Comment by Coop56** [comment]
This issue can be closed - there's a new ticket for @davisc to look into these errors on both the Composer Mapstory and Edit buttons.  @jonpmarino @reynalddizon

**Comment by jonpmarino** [comment]
Confirmed and closing!
